src/components/dashboard/Watchlist.js

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect } from "react";
+// tools
+import { connect } from "react-redux";
+import { ifDev } from "../../utils/removeAttribute.js";
+import { getWatchlistAction } from "../../store/actions/index.js";
+// children components
+import LoadingScreen from "../layout/LoadingScreen.js";
+import MovieCard from "../movies/MovieCard.js";
+
+function Watchlist({
+  userid,
+  isFetching,
+  watchlist,
+  watchlistError,
+  getWatchlistAction,
+}) {
+  console.log( watchlist)
+  useEffect(() => {
+    // Returns the most recent recommendations from the database
+    getWatchlistAction(userid);
+  }, []);
+
+  if (isFetching) return <LoadingScreen />;
+  else
+    return (
+      <div
+        className="watchlist-container"
+        data-test={ifDev("watchlist-component")}
+      >
+        <h2>Watchlist</h2>
+        <div className="movie-cards">
+          {watchlist.map((x, index) => {
+            let posterURI = x.poster_url;
+            let unsplashUrl =
+              "https://source.unsplash.com/collection/1736993/500x650";
```

Comment:
I would move this to the config file so that if it changes, you can easily locate it vs having a broken image on the app.

Reply:
Same for line 36

Reply:
I'll be updating this today to a proper place holder image.